docs/README.md

```diff
@@ -900,15 +900,132 @@ where the schema for `CreateProductInput` is
 
 ### 2. Events
 
+Events are **immutable facts** within your application's domain. They are the cornerstone of Booster because of its event-driven and event-sourced nature. Booster events are TypeScript classes decorated with `@Event`. An event class may looks like this:
+
+```typescript
+@Event
+export class EventName {
+  public constructor(readonly field1: SomeType,
+                     readonly field2: SomeOtherType,
+                     /* This event's entity ID must be present */) {}
+
+  public entityID(): UUID {
+    return /* the associated entity ID */
+  }
+}
+```
+
+Events and [entities](#4-entities-and-reducers) are closely related because entities represent the application's state after reducing the stream of events. All the events associated with the same entity are reduced together in the order they have occurred. If an event is related to an entity or not, it is domain-dependent. That's why Booster requires you to implement the `entityID` method. Entities and events are somehow coupled, but that coupling is limited to data. Take a look at this event:
```

Comment:
In the current implementation, it's a requirement that an event is related to one entity, and entities can only reduce their own events. I've changed the paragraph a little bit to reflect that:

```suggestion
Events and [entities](#4-entities-and-reducers) are intimately related. All events belong to one entity through the `entityID` field, and entities represent the application's state after reducing the stream of events. Indeed, an entity could be deleted and rebuilt from the events anytime, so an entity is just an aggregate representation of the same data that is stored in its events. It's guaranteed by Booster that all the events associated with an entity are reduced in the same order in which they have been stored. Take a look at this event:
```

Reply:
Agree with your change, Javi, but I would change the following: 
> All events belong to one entity through the `entityID` field...

"All events belong to one entity through the `entityID` **method**"

> Indeed, an entity could be deleted and rebuilt from the events anytime, so an entity is just an aggregate representation of the same data that is stored in its events.

Do you think it is a good idea to say "an entity could be _deleted_"? I think this could mislead people to think entities are stored someplace, instead of just a reduced stream of events (thinking this way will help them better to develop in Booster)

I possible rephrase could be:

"Indeed, an entity is just an aggregate representation of the same data that is stored in its events, so it can be rebuilt from the events anytime"